### PR TITLE
Add script for running dockers and collecting data

### PR DESCRIPTION
**Pull request by adamkral12**

### CODE REVIEW

File: docker_master.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+docker-compose up mongo postgres &
```
Comment (makovako) [inline docker_master.sh]: whats the differece to 'docker-compose up -d ...'?

File: docker_master.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```
Comment (makovako) [inline docker_master.sh:1]: add if running or hardcode command 'docker-compose down', i had it already running and bad things happened

File: docker-compose.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ services:
       - postgres
 #   output stat files to /dont/do/ in python, sync here into ./docker/pylo
     volumes:
-      - ./docker/pylo:/dont/do/
+      - ./output:/output
```
Comment (makovako) [inline docker-compose.yml:20]: It looks like this way doesn't work on linux, the directory is empty after app exits, i will search for other solutions, maybe just a different configuration of docker between mac and linux?

**Review by makovako — CHANGES_REQUESTED** [review]
Some problems to solve:
- put the containers down before testing, just in case
- some problems with linux

File: docker_master.sh (hunk 1 above)
Comment (adamkral12) [reply docker_master.sh]: probably the same, changed

File: docker_master.sh (hunk 2 above)
Comment (adamkral12) [reply docker_master.sh:1]: added

File: docker-compose.yml (hunk 3 above)
Comment (adamkral12) [reply docker-compose.yml:20]: maybe some permission issues?

**Review by makovako — APPROVED** [review]

**Pull request merged**